api/src/wfl/jdbc.clj

```diff
@@ -135,3 +137,20 @@
    `(do
       (log/info "JBDC SQL connection made:" (format-db ~db-spec) ~opts)
       (jdbc/get-connection ~db-spec ~opts))))
+
+;; Expertly copied and pasted from Stack Overflow:
+;; https://stackoverflow.com/a/25786990
+(extend-protocol clojure.java.jdbc/ISQLParameter
+  IPersistentVector
+  (set-parameter [v ^PreparedStatement stmt ^long i]
+    (let [conn          (.getConnection stmt)
+          meta          (.getParameterMetaData stmt)
+          [head & rest] (.getParameterTypeName meta i)]
+      (if-let [elem-type (when (= head \_) (apply str rest))]
+        (.setObject stmt i (.createArrayOf conn elem-type (to-array v)))
+        (.setObject stmt i v)))))
```

Comment:
Weird.  So `_` introduces a type name?

Reply:
> Cool. Was hoping I could watch this work in a test.
> Maybe `UserException` is enough of a name?

I'm just introducing the type - what would you have the test verify?
Rex has tests for the protocol extensions in his PR so I didn't bother as I'm hoping he'll merge soon. https://github.com/broadinstitute/wfl/pull/371